include/ck_tile/host/reference/reference_batched_elementwise.hpp

```diff
@@ -9,29 +9,31 @@
 
 namespace ck_tile {
 
-template <typename ADataType,
-          typename BDataType,
-          typename AccDataType,
-          typename CDataType,
+template <typename AccDataType,
+          typename ATensorView,
+          typename BTensorView,
+          typename CTensorView,
           typename AElementOp      = ck_tile::identity,
           typename BElementOp      = ck_tile::identity,
           typename BinaryElementOp = ck_tile::plus<AccDataType>>
-CK_TILE_HOST void reference_batched_elementwise(const HostTensor<ADataType>& a_b_m_n,
-                                                const HostTensor<BDataType>& b_b_m_n,
-                                                HostTensor<CDataType>& c_b_m_n,
+CK_TILE_HOST void reference_batched_elementwise(const ATensorView& a_b_m_n,
```

Comment:
Suggest use Tensor instead of TensorView. Because we have tensor_view.
Using TensorView easily make reader confuse.

Reply:
view is a non-owning type which is easy to copy like `std::span`, `std::string_view,`
use tensor view instead of tensor here remiind that the function itself is allowed to pass-by-value to other functions